Sources/Styleguide/DesignSystems/Align.swift

```diff
@@ -4,8 +4,10 @@ import Prelude
 extension Class {
   public enum align {
     public static let middle = CssSelector.class("align-middle")
+    public static let top = CssSelector.class("align-top")
   }
 }
 
 public let alignStyles =
   Class.align.middle % verticalAlign(.middle)
+    <> Class.align.top % verticalAlign(.top)
```

Comment:
we were missing this from our style guide. this causes looooots of snapshot failures, but not screenshot failures.